pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/BadgeCreateDTO.java

```diff
@@ -0,0 +1,85 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dto;
+
+import com.sun.istack.internal.NotNull;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.enums.ChallengeStatus;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * DTO for Badge creation request
+ *
+ * @author Michal Mokros 456442
+ */
+public class BadgeCreateDTO {
+
+    @NotNull
+    private Long trainerId;
+
+    @NotNull
+    private Long gymId;
+
+    @NotNull
+    private LocalDate date;
+
+    @NotNull
+    private ChallengeStatus status;
```

Comment:
When you create a badge, it is always WAITING_TO_ACCEPT - therefor this is unnecessary.